### ISSUE TITLE
Different graphics outputs for Editor and Notebook?

### ISSUE THREAD

**Issue by TrueShack** [spyder-ide/spyder-notebook#273]
## Is there a way to get different graphics outputs for Editor and Notebook?
I prefer graphics in an extra window in the Spyder editor, but because of the structure of Jupyter notebooks I would like them inline there after each cell (as it is standard). The Graphics backend in the Preferences changes both together. Can you change both settings separately? 

## Versions and main components

* Spyder Version: 4.1.1
* Python Version: 3.8.2
* Operating system: Win 10 x64

**Comment by goanpeca** [comment]
Hi @TrueShack, thanks for the suggestion. I think this does make sense for some use cases.

What do you think @spyder-ide/core-developers ?

**Comment by jitseniesen** [comment]
Yes, I agree with this. It should be possible to have separate settings for the console and the notebook.

**Comment by ccordoba12** [comment]
+1 from me too.

**Comment by jitseniesen** [comment]
Notebooks now always show plots inside the notebook. I am not sure where this changed, but I think with the release of Spyder 6. Showing the plots inside the notebook makes most sense to me, and it is what everybody who commented requested, so I am considering this issue fixed.
